forward_engineering/helpers/tableHelper.js

```diff
@@ -123,9 +123,10 @@ module.exports = app => {
 			iceberg: preSpace(iceberg && 'ICEBERG'),
 			dynamic: preSpace(dynamic && 'DYNAMIC'),
 			catalogSync: catalogSync ? `CATALOG_SYNC = '${catalogSync}'\n` : '',
-			storageSerializationPolicy: storageSerializationPolicy
-				? `STORAGE_SERIALIZATION_POLICY = ${toUpper(storageSerializationPolicy)}\n`
-				: '',
+			storageSerializationPolicy:
+				storageSerializationPolicy && !dynamic
```

Comment:
Need to investigate why it appears in all reversed tables. Will create a separate task for it.